doc/dev.md

```diff
@@ -98,9 +103,15 @@ Return generic "not found/failed" error message to user, log the original error.
 
 2. SCM errors
 Some of these can only be fixed by the user who is calling the method by interacting with UI elements (usually course teacher).
```

Comment:
I think this renders together with "SCM errors". Suggest to emphasize **SCM errors**